os/arch/arm/src/s5j/s5j_idle.c

```diff
@@ -83,8 +83,13 @@
 #ifdef CONFIG_PM
 static void up_idlepm(void)
 {
+#ifdef CONFIG_IDLE_PM
```

Comment:
As per the changes if flag CONFIG_S5J_IDLE_PM is enabled then this file will not get built.
then ther is no point in making changes to this file right ?

Reply:
yes , i will modify this